.github/workflows/ci.yml

```diff
@@ -40,15 +40,16 @@ jobs:
           python-version: ${{ matrix.python.action }}
           architecture: ${{ matrix.arch.action }}
 
-      - name: boots
+      - name: Install Poetry and Run
         run: |
-          python boots.py ensure --group build
+          python3 -m ensurepip
+          python3 -m pip install --upgrade pip
+          python3 -m pip install poetry
+          poetry install
 
-      - name: Package executable
+      - name: run pyinstaller
```

Comment:
Is changing from "Package executable" to "run pyinstaller" necessary? I suppose you could say "Package executable with pyinstaller"

Reply:
will revert, not sure when i did this.